### PR TITLE
work for terraform 0.14

### PR DESCRIPTION
**Pull request by dmattia**

### CODE REVIEW

**Comment by joe-niland** [comment]
@dmattia could you please update the context.tf file in the examples directory too?

**Comment by dmattia** [comment]
Done 😄 

@joe-niland

File: context.tf [hunk 1]
```diff
@@ -19,7 +19,7 @@
 #
 
 module "this" {
-  source = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.19.2"
+  source = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.22.0"
```
Comment (aknysh) [inline context.tf:22]: @dmattia 
Please use this format 
```
  source  = "cloudposse/label/null"
  version = "0.22.0"
```

we are updating all modules now.
thanks

File: examples/complete/context.tf [hunk 2]
```diff
@@ -19,7 +19,7 @@
 #
 
 module "this" {
-  source = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.19.2"
+  source = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.22.0"
```
Comment (aknysh) [inline examples/complete/context.tf:22]: source  = "cloudposse/label/null"
version = "0.22.0"

**Review by aknysh — CHANGES_REQUESTED** [review]
please see comments

**Comment by dmattia** [comment]
Updated the source type 👍

**Comment by aknysh** [comment]
@dmattia  thanks!
Please fix one more issue :)  (sorry for this, we just updating all modules and all tests to the latest standards).

To fix this error 
```
not ok - check if terraform providers have explicit source locations for TF =>0.13
```

please update all the providers (in the module and in the `examples/complete` test) to use this format 

```
terraform {
  required_version = ">= 0.12.0"

  required_providers {
    aws = {
      source  = "hashicorp/aws"
      version = ">= 3.0"
    }
    template = {
      source  = "hashicorp/template"
      version = ">= 2.0"
    }
    local = {
      source  = "hashicorp/local"
      version = ">= 1.3"
    }
  }
}
```

**Comment by dmattia** [comment]
perfect, just updated @aknysh

**Review by aknysh — APPROVED** [review]
thanks @dmattia

**Pull request merged**